[{"lineNumber": 9, "col_offset": 8, "nodeName": "model", "type": "Any"}, {"lineNumber": 9, "col_offset": 16, "nodeName": "PatientFactory", "type": "Any"}, {"lineNumber": 21, "col_offset": 15, "nodeName": "Exception", "type": "Type[Exception]"}, {"lineNumber": 10, "col_offset": 15, "nodeName": "str", "type": "Type[str]"}, {"lineNumber": 10, "col_offset": 19, "nodeName": "model", "type": "Any"}, {"lineNumber": 10, "col_offset": 29, "nodeName": "format", "type": "Callable[..., str]"}, {"lineNumber": 11, "col_offset": 12, "nodeName": "__name__", "type": "Any"}, {"lineNumber": 11, "col_offset": 38, "nodeName": "name", "type": "Any"}, {"lineNumber": 20, "col_offset": 12, "nodeName": "factory", "type": "Any"}, {"lineNumber": 11, "col_offset": 12, "nodeName": "__class__", "type": "Any"}, {"lineNumber": 11, "col_offset": 38, "nodeName": "model", "type": "Any"}, {"lineNumber": 22, "col_offset": 12, "nodeName": "fail", "type": "Any"}, {"lineNumber": 11, "col_offset": 12, "nodeName": "model", "type": "Any"}, {"lineNumber": 22, "col_offset": 12, "nodeName": "pytest", "type": "Any"}]